run_pipeline_on_gadi_script.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+  
+#PBS -P er01 
+#PBS -N Somatic-shortV
+#PBS -l walltime=02:00:00
+#PBS -l ncpus=1
+#PBS -l mem=60GB
+#PBS -W umask=022
+#PBS -q copyq
+#PBS -e Somatic-shortV-nf.e
+#PBS -o Somatic-shortV-nf.o
+#PBS -l wd
+#PBS -l storage=scratch/er01+gdata/er01
+#PBS -l jobfs=10GB
+
+# Load singularity and nextflow modules
+module load nextflow/22.04.3
+module load singularity
+
+export NXF_SINGULARITY_CACHEDIR=/scratch/$PROJECT/$(whoami)/singularity
+
+# Fill in these variables for your run
+samples=/scratch/er01/ndes8648/pipeline_work/nextflow/INFRA-83-Somatic-ShortV/test_files_for_Georgie/samples.csv
+ponvcf=/scratch/er01/ndes8648/pipeline_work/nextflow/INFRA-83-Somatic-ShortV/test_files_for_Georgie/pon.vcf.gz
+ref=/g/data/er01/SIH-HPC-WGS/Reference/hs38DH.fasta
+small_exac_common=/g/data/er01/SIH-HPC-WGS/Reference/gatk-best-practices/somatic-hg38/small_exac_common_3.hg38.vcf.gz
+intervalList_path=/scratch/er01/ndes8648/pipeline_work/nextflow/INFRA-83-Somatic-ShortV/test_files_for_Georgie/interval_files
+outDir=results
+
+
+# Select one of the following options
+	# (A)  
+	# - Generate optimised number of intervals based on your chosen genome and its size
+	# - Please do the following:
+		# - Hash to inactivate the `number_of_intervals` variable below
+		# - You will also need to provide path to reference genome .dict file
+			dict=/g/data/er01/SIH-HPC-WGS/Reference/hs38DH.dict
+
+	# OR 
+
+	# (B) 
+	# Generate an arbitrary number of intervals between 2 and 100 based on your system resources
+		#number_of_intervals=50
```

Comment:
None of this is necessary and doesn't actually make much sense given your required inputs specified in `main.nf`. You can opt to define a variable if it is being applied otherwise it does not need to be included. Remove this whole section and instead add `dict=` (which you've actually mandated as a required parameter) and `number_of_intervals=` with default values and describe clearly in which situations and how to apply each variable in the relevant section of the README. 

Additionally, a comment like 'generate an arbirtary number of intervals between 2 and 100 based on your system resources' needs to be explained. How would I know what is an appropriate number and what you mean by system resources?